### ISSUE TITLE
Feedback for “Setup Ents in an existing Convex project”

### ISSUE THREAD

**Issue by cathedral-on-the-moon** [xixixao/convex-ents#31]
I couldn’t really understand how to use Pick (so that I can use ctx.db for part of my schema). A dedicated section in the docs with examples  would help.

**Comment by xixixao** [comment]
@cathedral-on-the-moon 

> I couldn’t really understand how to use Pick

What made it hard to understand? Did you try to follow the doc and ran into some errors?

**Comment by cathedral-on-the-moon** [comment]
When I originally created this GitHub issue, I did not see the example of Pick in this section: https://labs.convex.dev/convex-ents/setup/config#incremental-adoption-restrict-ctxdb
Let me  close this issue for now. Let me try again if I need to keep ctx.db
